### DISCUSSION TITLE
Slack authentication error

### DISCUSSION THREAD

**Discussion by TwoBrake** [outline/outline#4970] [Self Hosting] [answered]
When I setup the Slack authentication, it errors after logging in with Slack. Please let me know if there is any fix to this issue.

**To Reproduce**
1. Click on the login with Slack button.
2. It will redirect you to slack and error.

Here is the error:
```
redirect_uri did not match any configured URIs. Passed URI: https://docs.host.com/auth/slack.callback
```
Thank you in advance.

**Comment by TwoBrake** [comment, marked as answer]
I figured it out, it had something to do with the characters being incorrect.

**Reply by IshuaWang** [reply]
hi, what's the problem? I have met the same issue.
